### PR TITLE
Add meeting minutes from 05-November-2024 meeting

### PR DESCRIPTION
**Pull request by jgust**
Add meeting minutes for yesterday's meeting

### CODE REVIEW

**Review by JoelMarcey — APPROVED** [review]

File: subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (PLeVasseur) [inline subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md:2]: I think this makes more sense. No need to have both an `agenda.md` and `minutes.md`. Could you rename this to `minutes.md` to signify what it is now?

File: subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md [hunk 2]
```diff
@@ -14,25 +14,85 @@ Supplemental material to the agenda can be found on the [GitHub repo](https://gi
...
+* Meeting introduction and check-in
+* Meeting minutes from last meeting were quickly reviewed and accepted
+
+### Contributor Expectations \- Online discussion
+
+* Contributor expectations doc on GitHub were updated to use the Rust Foundation’s CoC
+* Discussion about how to phrase the sentence of who can be a contributor and how much should they contribute.
+* Everyone who joins in should be contributed
+* General consensus that the wording made it sound like a unnecessary barrier.
+* Pete takes an action to remove the current contributor document from the PR and recreate one in a separate PR
```
Comment (PLeVasseur) [inline subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md:57]: Move up under tasks?

File: subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md [hunk 3]
```diff
@@ -14,25 +14,85 @@ Supplemental material to the agenda can be found on the [GitHub repo](https://gi
...
+
+* Choice of collaboration tool TBD due to various network restriction at companies
+
+### MISRA \+ Rust
+
+* MISRA will publish a whitepaper detailing _possible_ applicability of the existing C rules to Rust (without changes, except to Category; no new rules,
+  not a freestanding guidelines document), but probably won't enthrone a WG any time in the near future due to lack of membership and redundancy w.r.t this Subcommittee;
+* "Iron Carbide" is an IP-neutral take on the applicable rules, plus a few original rules to plug really big gaps; this aims to be a document mirroring
+  MISRA's structure w.r.t compliance etc. and rule descriptions; MISRA is _not_ currently planning to publish this so we'd like to hand it over to the Subcommittee for review instead
+* The Subcommittee can then take "moral ownership of it (with a big "whatever that means" to be decided later)
```
Comment (PLeVasseur) [inline subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md:86]:
```suggestion
* The Subcommittee can then take "moral ownership" of it (with a big "whatever that means" to be decided later)
```

**Review by PLeVasseur — COMMENTED** [review]
Thanks @jgust! Left a few suggestions.

File: subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md (hunk 1 above)
Comment (jgust) [reply subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md:2]: sure!

File: subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md (hunk 2 above)
Comment (jgust) [reply subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md:57]: Good point!

File: subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md (hunk 3 above)
Comment (jgust) [reply subcommittee/coding-guidelines/meetings/05-November-2024/agenda.md:86]: Done!

**Review by ah801 — APPROVED** [review]

**Review by PLeVasseur — APPROVED** [review]
Thanks to our notetaker from last time @jgust for taking them and formatting them here!

LGTM

**Comment by jgust** [comment]
Rebase done!

**Pull request merged**